webapp/src/modules/ui/browse/selectors.ts

```diff
@@ -90,3 +92,21 @@ export const getOnSaleElements = createSelector<
   ReturnType<typeof getOnSaleNFTs>,
   OnSaleElement[]
 >(getOnSaleItems, getOnSaleNFTs, (items, nfts) => [...items, ...nfts])
+
+
+export const getClaimingBackState = (state: RootState, nft: NFT): boolean => {
+  const userAddress = getAddress(state);
+
+  const transactionsClainLand = getTransactionsByType(state, userAddress!, CLAIM_LAND_TRANSACTION_SUBMITTED)
```

Comment:
Following the name of the transaction:
```suggestion
  const claimLandTransactions = getTransactionsByType(state, userAddress!, CLAIM_LAND_TRANSACTION_SUBMITTED)
```

Reply:
We must consider the case where the `userAddress` is undefined or null, as it could happen that the user is not logged in.
We can return false for this case.